### ISSUE TITLE
Auto32k doesn't work

### ISSUE THREAD

**Issue by omninope** [zeroeightysix/KAMI#238]
**Describe the bug**
The module Auto32k fails and give some errors in chat if the option Debug Messages is enabled, otherwise nothing appears to happen.
Tested with and without other modules enabled, no difference.

**To Reproduce**
1. Enable Auto32k;
2. See the errors in chat.

**Expected behavior**
It should work, at least I hope this is the intention.

**Screenshots**
![2020-09-15_22 39 07](https://user-images.githubusercontent.com/69328977/93282463-02244600-f7be-11ea-9581-2abfab4e3e15.png)

**Environment**
Did you build KAMI yourself? If so, on what commit? If not, which release did you download?
Yes, commit: 40a8068b67ef0f7e749fafccda9c1bdeaf86ea47

What operating operating system?
`uname -a` gives: `Linux debian 5.7.0-3-amd64 #1 SMP Debian 5.7.17-1 (2020-08-23) x86_64 GNU/Linux`.

What version of java?
`java -version` gives: `Java(TM) SE Runtime Environment (build 1.8.0_221-b11)`.

**Additional context**
No.

**Comment by dominikaaaa** [comment]
Try reading the error message

**Comment by JerryWester** [comment]
Isn't Auto32k only really a workaround for an exploit that was patched on 2b2t?

**Comment by dominikaaaa** [comment]
Uh, no?

**Comment by dominikaaaa** [comment]
This is an example of the same module on 1.12.2 https://www.youtube.com/watch?v=bXkXQO_lLfs

**Comment by omninope** [comment]
> Try reading the error message

There is plenty of space arround me in the screenshot, I don't think there isn't any lack "valid positions" to place, and i tested many times in different situations and it never worked.
I think shoud have added this comment when i opened the issue.

**Comment by dominikaaaa** [comment]
A valid position would include an enemy

**Comment by omninope** [comment]
So a enemy is **required** in order to place the 32k?
If that's the case, the phrase "No enemy nearby, placing at first valid position." feels kinda misleading, or maybe is the fact that my english is bad and I probably can't comprehend it.

**Comment by dominikaaaa** [comment]
Yes, that is why I linked you the video. I guess it is misleading, I don't understand why it says that.
